content/kapp-controller/docs/latest/package-authoring.md

```diff
@@ -137,6 +137,10 @@ spec:
   - demo
 ```
 
+When creating this CR, the api will validate that the Package's name is a fully
```

Comment:
It might be helpful to explicitly call out that the name above is a valid name: `An example of a fully qualified is shown for the Package CR above: simple-app.corp.com`.